pallets/frequency-tx-payment/src/lib.rs

```diff
@@ -205,6 +219,33 @@ pub mod pallet {
 
 			call.dispatch(origin)
 		}
+
+		#[pallet::call_index(1)]
+		#[pallet::weight({
+		let dispatch_infos = calls.iter().map(|call| call.get_dispatch_info()).collect::<Vec<_>>();
+		let dispatch_weight = dispatch_infos.iter()
```

Comment:
I think we can create an overhead_weight function since the computation is done in both `pay_with_capacity` and `pay_with_capacity_batch_all` and we leave that big comment at the top of it.

```
/// Dispatch the given call as a sub_type of pay_with_capacity. Calls dispatched in this
		/// fashion, if allowed, will pay with Capacity
		// The weight calculation is a temporary adjustment because overhead benchmarks do not account
		// for capacity calls.  We count reads and writes for a pay_with_capacity call,
		// then subtract one of each for regular transactions since overhead benchmarks account for these.
		//	 Storage: Msa PublicKeyToMsaId (r:1)
		//   Storage: Capacity CapacityLedger(r:1, w:2)
		//   Storage: Capacity CurrentEpoch(r:1) ? maybe cached in on_initialize
		//   Storage: System Account(r:1)
		//   Total (r: 4-1=3, w: 2-1=1)
```